diff --git a/config/win/BUILD.gn b/config/win/BUILD.gn
index c51ad032a..340c82826 100644
--- a/config/win/BUILD.gn
+++ b/config/win/BUILD.gn
@@ -471,8 +471,8 @@ config("default_crt") {
       # CRT in Visual Studio releases related to Windows store applications.
       configs = [ ":dynamic_crt" ]
     } else {
-      # Desktop Windows: static CRT.
-      configs = [ ":static_crt" ]
+      # Desktop Windows: dynamic CRT.
+      configs = [ ":dynamic_crt" ]
     }
   }
 }
@@ -501,7 +501,7 @@ config("release_crt") {
       ldflags = [ "/DEFAULTLIB:msvcprt.lib" ]
     }
   } else {
-    cflags = [ "/MT" ]
+    cflags = [ "/MD" ]
 
     if (rust_prebuilt_stdlib) {
       rustflags = [ "-Ctarget-feature=+crt-static" ]
